features/user_sessions.feature

```diff
@@ -66,9 +66,22 @@ Feature: User sessions
       | contact_name | email            | password  | password_updated_at | approval_status |
       | Jane         | jane@example.com | test1234! | 2017-01-01          | approved        |
     And a clear email queue
-    And the date is 2017-01-02
+    And the time is 2017-03-30 23:55
     And I log in with email "jane@example.com" and password "test1234!"
-    And the date becomes 2017-04-15
+    And the time is 2017-03-31 00:20
     And I follow "Add Site"
+    And the time is 2017-03-31 01:10
```

Comment:
This line is not necessary for this test.

Reply:
Its to show that the password now expired. So the login was done at 03/30 password expires 03/31 because its 90 days since the password was set.

Reply:
I missed it was already there. Gotcha thanks